### DISCUSSION TITLE
Feedback på ordlistan

### DISCUSSION THREAD

**Discussion by araskazemi** [diggsweden/ena-authorization#19] [General]
Anvisningstjänst eller hänvisningstjänst?
Glöms inte maskinidentiteter bort?

**Comment by rob1sun** [comment]
claims: kanske utveckla lite. Ett attribut blir en del av ett claim i en assertion.

**Reply by araskazemi** [reply]
Ser vi claim som synonym med assertion (svenska intyg)? @rob1sun

**Reply by araskazemi** [reply]
Eller är Calims synonym med Attribut, vilka ingår i en token eller assertion? :-)

**Reply by araskazemi** [reply]
Nu finns ett förslag på claims och attribut i en gren som ska hantera uppdatering av ordlistan https://github.com/diggsweden/ena-authorization/blob/ordlista-remiss-2024-aug/introduktion/ordlista.md#grundidentifiering

**Comment by araskazemi** [comment]
SAML, OAuth och OpenID Connect bör förklaras.
